src/common/enums/enums.ts

```diff
@@ -0,0 +1,3 @@
+export { AuthorName } from './author/author';
+export { CodeLanguage } from './code/code';
+export { ContactIcon } from './icon/icon';
```

Comment:
modules should be more common, like CodeLanguage and ContactIcon can be in ui, AuthorName in lectures